flipperdevices/flipperzero-firmware
# Documentation: fixed outdated naming

Changed because of outdated naming in assets/dolphin/ReadMe. `essential -> blocking`

# What's new

- Correct file naming in ReadMe's (`assets/dolphin/ReadMe.md` and `assets/ReadMe.md`)

# Verification 

- Check naming of files in path `assets/dolphin/` and `assets/`
or 
- Check how `assets/SConscript` names files and folders

# Checklist (For Reviewer)
